Gopkg.lock

```diff
@@ -1069,6 +1087,7 @@
     "k8s.io/apimachinery/pkg/runtime/serializer",
     "k8s.io/apimachinery/pkg/types",
     "k8s.io/apimachinery/pkg/util/clock",
+    "k8s.io/apimachinery/pkg/util/diff",
```

Comment:
any idea why this got added? i don't see any refs to it

Reply:
Absolutely not :)